src/components/Navbar.css

```diff
@@ -0,0 +1,114 @@
+nav {
```

Comment:
Ideally, you should have replaced `Navbar.scss` with `Navbar.css` in a separate commit.
As this commit deals with only adding `Project Page`.

I have created a separate commit. You can learn about this if you want. (How we perform this action).